glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
I am going to fix the the nl code so runner can pass the kernel status about the nl command. So for TCMU_CMD_ADDED_DEVICE, runner will be able to return failure if the handler's open() fails. That error will then be propagated through the kernel so lio can clean itself up, and to rtslib and whatever tools is using that (targetcli, gwcli, gluster's tool) so they can return an error to the user.

So I am going to just kill check_config and in the open calls, make sure the same checks are done if needed.